examples/MQ/pixelSimSplit/src/CMakeLists.txt

```diff
@@ -0,0 +1,80 @@
+ ################################################################################
+ #    Copyright (C) 2014 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH    #
+ #                                                                              #
+ #              This software is distributed under the terms of the             # 
+ #              GNU Lesser General Public Licence (LGPL) version 3,             #  
+ #                  copied verbatim in the file "LICENSE"                       #
+ ################################################################################
+# Create a library called "libPixel" which includes the source files given in
+# the array .
+# The extension is already found.  Any number of sources could be listed here.
+
+set(INCLUDE_DIRECTORIES
+  ${BASE_INCLUDE_DIRECTORIES}
+  ${FairMQ_INCDIR}/fairmq/options
+  ${CMAKE_SOURCE_DIR}/base/MQ/policies/Serialization
+  ${CMAKE_SOURCE_DIR}/examples/MQ/pixelDetector/src
+  ${CMAKE_SOURCE_DIR}/examples/MQ/pixelDetector/src/devices
+  ${CMAKE_SOURCE_DIR}/examples/MQ/pixelSimSplit/src
+  ${CMAKE_SOURCE_DIR}/examples/MQ/pixelSimSplit/src/devices
+  ${CMAKE_SOURCE_DIR}/examples/common/mcstack
+  ${FairLogger_INCDIR}
+)
+
+
+Set(SYSTEM_INCLUDE_DIRECTORIES
+    ${SYSTEM_INCLUDE_DIRECTORIES}
+    ${ZeroMQ_INCLUDE_DIR}
```

Comment:
You don't need `${ZeroMQ_INCLUDE_DIR}`, which also makes the whole `Set(SYSTEM_INCLUDE_DIRECTORIES` unnecessary, including the `Include_Directories(SYSTEM ${SYSTEM_INCLUDE_DIRECTORIES})`.